content/04.model-complexity.md

```diff
@@ -1,42 +1,45 @@
-Below, we describe some examples, including bootstrapping sample data, regularization methods to protect from overfitting, and ensembling techniques to minimize misinterpretation of the data. 
+In addition to improving the input data, methodological considerations can also help address the challenges posed by small datasets with high label-uncertainty in rare diseases.
+Below, we describe some examples, including bootstrapping sample data, regularization methods to protect from overfitting, and ensembling techniques to minimize misinterpretation of the data.
 
 #### Bootstrapping
 
 Bootstrapping is a powerful statistical technique that can be used to estimate population values from datasets of limited sample size by resampling the data [@doi:10.1080/01621459.1997.10474007].
 Bootstrap based techniques are used in conjunction with various learning methods to find the most informative models given a specific dataset (e.g. bootstrap aggregating or _bagging_ used in random forests [@doi:10.1023/A:1010933404324; @doi:10.1198/0003130043277], bootstrap in neural networks [@doi:10/c8xpqz], or regression models [@doi:10.1016/j.neucom.2004.11.017; @doi:10.1002/sim.4780111607]).
-Bootstrapping can also be used to enhance the information content of rare disease datasets and generate confidence intervals for the model predictions as demonstrated by Banerjee et al [@doi:10.3390/genes11020226].
+Bootstrapping can also be used to enhance the information content of rare disease datasets and generate confidence intervals for the model predictions [@doi:10.3390/genes11020226].
+<!-- TODO: I would remove the following two sentences. It wasn't clear to me how it went above and beyond what was already stated here.
 In this study, bootstrapping the training sample without replacement simulated generation of separate datasets that helped expose the learning models (random forests) to the incomplete nature of the data.
-Such bootstrapping of the training data in addition to that included in the model (bagging) helped generate a distribution (and confidence intervals) of the importance scores of the predictive features selected by the model.
+Such bootstrapping of the training data in addition to that included in the model (bagging) helped generate a distribution (and confidence intervals) of the importance scores of the predictive features selected by the model. -->
 
 #### Regularization
 
-Another common strategy for handling the paucity of data in rare disease is to aggregate data from multiple studies or time points to produce a more comprehensive dataset. 
+<!-- TODO: I think of regularization as a way to get simpler models. I'm not sure if the properties described in the first paragraph of this section are things that would be improved by simpler models. It feels like these are more like technical biases that simpler models might be just as good at picking up on. -->
```

Comment:
yeah, this paragraph is protraying regularization in the context of minimizing overfitting to reduce model complexity. While I think the end result is making simpler models, I agree that has not come across clearly here. 

Would you be able to point me to some literature that talks about regularization in the context of making simpler models? I think I came across mostly mechanistic papers.

Reply:
I am planning to write an initial draft of the putting it all together section (#111), which we expect to include discussion of regularization. Do we think it would be helpful for me to draft that section _first_ and then circle back and address some of the comments on this section? My rationale is that it will be important for this section on regularization highlight specific points that are necessary for contextualizing MultiPLIER and DeepProfile.

Reply:
tracked in #112